server/validators/validateUserController.js

```diff
@@ -0,0 +1,91 @@
+const Users = require("../models/Users");
+
+exports.validateUniqueUrl = async function (req) {
+  // init
+  let isValid = false;
+  let message = "";
+
+  // Check that url is unique
+  urls = await Users.find({ calendarUrl: req });
+
+  if (urls.length > 0) {
+    message = "this url is not unique";
+    return { isValid, message };
+  }
+
+  isValid = true;
+  message = "this url is unique";
+  return { isValid, message };
+};
+
+exports.validateUserInfo = async function (req) {
+  //init
+  let isValid = false;
+  let message = "";
+  const name = req.body.name;
+  const avatarUrl = req.body.avatarUrl;
+  const timeZone = req.body.timeZone;
+  const availableHoursFrom = req.body.availableHoursFrom;
+  const availableHoursTo = req.body.availableHoursTo;
+  const availableDays = req.body.availableDays;
+  const calendarUrl = req.body.calendarUrl;
+
+  // Check that name is not null
+  if (!name) {
+    isValid = false;
+    message = "name shouldn't be an empty";
+    return { isValid, message };
+  }
+  // Check that is not null
+  if (!avatarUrl) {
+    isValid = false;
```

Comment:
The code 
if (!<field>)
 isValid = false;
 message = "<field> shouldn't be empty";
 return { isValid, message };
is repeated in some ways, you can combine it in a function like I did in appointmentController. This is not necessary.
I think the code looks good. One thing I would recommend is run it through postman to see if your controller is working for all scenarios and is bug free. I can help with that if you want.